entity-utils/src/main/java/eu/europeana/entity/utils/EntityUtils.java

```diff
@@ -48,4 +48,21 @@ public static List<String> updateBaseUrlInIds(List<String> entityUris, String da
         }
         return updatedUris;
     }
+    
+    public static String validateUriAndRemoveQuotes(String uri) {
+        boolean startsWithDoubleQuotes = uri.startsWith("\"");
+        boolean endsWithDoubleQuotes = uri.endsWith("\"");
+        
+        if(startsWithDoubleQuotes && endsWithDoubleQuotes) {
```

Comment:
what happens when URL starts with " or ends with "?

Reply:
we can be fault tolerant an remove the quote at the start and/or the end of the URL

Reply:
Please use commons lang StringUtils.removeStart() and removeEnd() methods